shared/locales/en.js

```diff
@@ -389,6 +389,9 @@ Your accounts' balances:`,
                 new_operations: 'New operations imported during this period:',
                 no_new_operations: 'No new operations have been imported during that period.'
             }
+        },
+        notification: {
+            new_operation: `Kresus: %{operationsCount} new transaction(s) imported`
```

Comment:
instead of the `(s)`, can you try one of polyglot's features: `||||` is a placeholder for a position that needs auto pluralization:

```
new_operation: `Kresus: %{operationsCount} new transaction |||| imported`
```